### PR TITLE
Updated the todo-sql example readme

### PR DESCRIPTION
**Pull request by aish-where-ya**
# Feature or Problem
This updated readme provides specific instructions and screenshots for running the Todo-sql example with Postgres.

# Related Issues
N/A

# Release Information
Next Release

# Consumer Impact
No breaking changes. Users trying out the example have updated dependencies of the http server and other packages.

# Testing
Tested on platform(s)
- [ ] x86_64-linux
- [x] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

## Platforms
Built on platform(s)
- [ ] x86_64-linux
- [x] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

## Unit Test(s)
N/A

## Acceptance or Integration
Dependencies for integration tests may have to be updated.

## Manual Verification
Tested on local machine. Reviewers are welcome to test it on their machine.
Additional troubleshooting steps have been added to help users run on their specific machines.

### CODE REVIEW

File: actor/todo-sql/README.md [hunk 1]
```diff
@@ -9,26 +9,82 @@ and the logging capability (`wasmcloud:logging`).
 As usual, it is worth noting that this actor does _not_ know where its HTTP server comes from,
 nor does it know which relational database implementation the host runtime has provided.
 
-## To generate self-signed TLS certificates
+## Step 1 : Getting Started
+
+- Clone the repo using either HTTPS or ober SSH.
```
Comment (connorsmith256) [inline actor/todo-sql/README.md:14]: Supernit: typo in `ober`

File: actor/todo-sql/README.md [hunk 2]
```diff
@@ -9,26 +9,82 @@ and the logging capability (`wasmcloud:logging`).
...
+postgres[ql]://[username[:password]@][host[:port],]/database[?parameter_list]
+
+\_____________/\____________________/\____________/\_______/\_______________/
+     |                   |                  |          |            |
+     |- schema           |- userspec        |          |            |- parameter list
+                                            |          |
+                                            |          |- database name
+                                            |
+                                            |- hostspec
+```
```
Comment (connorsmith256) [inline actor/todo-sql/README.md:69]: Nice!

File: actor/todo-sql/README.md [hunk 3]
```diff
@@ -9,26 +9,82 @@ and the logging capability (`wasmcloud:logging`).
...
 
-- If you want to change from the default https port 9000, edit http-linkdef.json to change the port in the address line and also in the CORS origins
-- set up postgres database with a user that has write access (can create tables, insert, and select)
-- edit sql-linkdef.json and update the uri field with the credentials and host/port
-- make sure nats and the wasmcloud host are running
-- run `make clean-start` to push the actor, start the providers, and link neverything
-- Open your browser to https://localhost:9000/", and accept the warning from the browser that the server is using a self-signed certificate.
+## Step 3 -  Running WasmCloud and NATS
```
Comment (connorsmith256) [inline actor/todo-sql/README.md:71]: Supernit: we prefer to spell wasmCloud with a lowercase 'w'

File: actor/todo-sql/README.md [hunk 4]
```diff
@@ -9,26 +9,82 @@ and the logging capability (`wasmcloud:logging`).
...
+- Congratulations you have run the postgres example successfully.
 
-- visit "https://localhost:9000/test/" to view the todo test page and press the green button"
\ No newline at end of file
+## Troubleshooting
+- Make sure all the ports that are being used are open : 4000, 5000, 5432 and 9000.
+- Docker should have permission to access the examples folder and its subdirectories as it needs access to the `create-tables.sql file`.
+- Check the paths in `http-linkdef.json` and uri in `sql-linkdef.json`. Check if your postgres database is working in the specific docker container and that the correct user has all the necessary permissions.
+- `wash up` uses port 4000 as of now. If the host is unable to start, inspect this port and kill the necessary processes before restarting. `wash down` will gracefully shut down the host as well.
+- If you see a `base64: invalid argument -w0` in your logs, change `base64 -w0` to `base64 -i` in your Makefile.
```
Comment (connorsmith256) [inline actor/todo-sql/README.md:90]: Ideally the makefile should work on different platforms. I _think_ this change to the makefile would work with both mac and GNU versions of base64:
```bash
base64 -w0 sql-linkdef.json 2>/dev/null || base64 -i sql-linkdef.json 2>/dev/null
```

It's a bit hacky, but better than assuming the user has a specific version installed

**Review by connorsmith256 — CHANGES_REQUESTED** [review]
Just a couple nits and one change I think we should try

File: actor/todo-sql/README.md (hunk 4 above)
Comment (aish-where-ya) [reply actor/todo-sql/README.md]: Yes this works. Changed it in the makefile.

**Review by connorsmith256 — APPROVED** [review]

**Review by thomastaylor312 — APPROVED** [review]

**Pull request merged**